tests/review-snapshot.test.ts

```
@@ -43,7 +49,7 @@ function judgmentDayBudget(): ReviewBudgetV1 {
 }
 
 function snapshotGit(snapshot: SnapshotV1, ...args: string[]): string {
-	return execFileSync("git", args, {
+	return execFileSync("git", args, { env: sandboxGitEnv(), 
```

Biome (2.5.8)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)